app/client/components/wizardRouterAdapter.tsx

```diff
@@ -65,32 +53,23 @@ export const ReturnToAccountOverviewButton = () => (
   />
 );
 
-const RootComponent = (props: RootComponentProps) => (
-  <>
-    <PageContainer fullWidth>{props.thisStageChildren}</PageContainer>
-    {props.extraFooterComponents}
-  </>
-);
+interface CurrentStepProps {
+  path: string;
```

Comment:
Let's constrain this as a string literal type
```suggestion
  path: "/";
```

Reply:
to help reinforce the idea that this component is a holder for 'the content of this step'